### PR TITLE
[Squid] Add NFS scale tests

### PR DESCRIPTION
**Pull request by vamahaja**

### CODE REVIEW

**Review by msainiRedhat — CHANGES_REQUESTED** [review]

File: conf/reef/nfs/tier-0.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
+globals:
```
[the remark this answers is not in the document]
Comment (vamahaja) [reply conf/reef/nfs/tier-0.yaml]: Using existing conf - https://github.com/red-hat-storage/cephci/blob/master/conf/squid/nfs/1admin-7node-3client.yaml

**Review by msainiRedhat — APPROVED** [review]
LGTM

**Review by MohitBis — APPROVED** [review]
LGTM

**Review by vdas-redhat — APPROVED** [review]
LGTM

**Pull request merged**